tests/test_clean.py

```diff
@@ -1,96 +1,210 @@
+import os
+
 from html5lib.filters.base import Filter
 import pytest
 
-import bleach
+from bleach import clean
 from bleach.sanitizer import Cleaner
 
 
-def test_empty():
-    assert bleach.clean('') == ''
+def test_clean_idempotent():
+    """Make sure that applying the filter twice doesn't change anything."""
+    dirty = '<span>invalid & </span> < extra http://link.com<em>'
+    assert clean(clean(dirty)) == clean(dirty)
```

Comment:
This is the same test--I moved it to the top figuring "this is testing a property of `clean`".